components/02-molecules/page-title/page-title.stories.js

```diff
@@ -16,7 +16,7 @@ export default {
       type: 'string',
       defaultValue: `<span>By Charlyn Paradis</span>${dateTimeTwig({
         date_time__start: '2022-01-25',
-        date_time__format: 'date',
```

Comment:
I'm not sure why you changed these... In the designs it looks like it's using the `date` format (without the day of the week) to me. However, I think the event meta looks like it should be using `day__full`, so that could be changed in `yds-event-meta.twig`.

You can see the various date formats in `yds-date-time.twig` starting on line 19, if you're curious what they are.